metrics/platform-metrics-reporting/src/main/java/ddf/metrics/reporting/internal/rrd4j/RrdMetricsRetriever.java

```diff
@@ -478,11 +496,131 @@ public OutputStream createXlsReport(List<String> metricNames, String metricsDir,
         return bos;
     }
 
+    private void createSummary(Workbook wb, List<String> metricNames, String metricsDir,
+            long startTime, long endTime, SUMMARY_INTERVALS summaryInterval)
+            throws IOException, MetricsGraphException {
+        // convert seconds to milliseconds
```

Comment:
I actually read that API too, completely forgot it was there. I'll fix mine and all the existing ones as well.